src/functions/getItem.ts

```diff
@@ -0,0 +1,10 @@
+import { IItem } from "@/types/nodeTypes";
+
+export function getItem(item: IItem, path: string[]): IItem {
+  const arrIndex = Number(path.shift()) - 1;
+  if (path.length === 0) {
```

Comment:
if (!path.length) {

Reply:
and if you're using return no need to use else block

Reply:
fixed